pkg/network/protocols/postgres/testutils.go

```diff
@@ -158,3 +162,67 @@ func runTimedQuery(callback func(context.Context, ...interface{}) (sql.Result, e
 	_, err := callback(ctx)
 	return err
 }
+
+var (
+	mapTypesToZero = map[ebpf.MapType]struct{}{
+		ebpf.PerCPUArray: {},
+		ebpf.Array:       {},
+		ebpf.PerCPUHash:  {},
+	}
+)
+
+// CleanPostgresMaps deletes all entries from the postgres maps. Test utility to allow reusing USM instance without caring
+// over previous data.
+func CleanPostgresMaps(t *testing.T) {
```

Comment:
There's a nearly identical function in `kafka/testutils.go`, can we refactor and reuse it?

Reply:
done, good callout